src/nomad_material_processing/__init__.py

```diff
@@ -123,6 +244,154 @@ class Parallelepiped(Geometry):
     )
 
 
+class TruncatedCone(Geometry):
+    """
+    Class autogenerated from yaml schema.
```

Comment:
Just saw this... I'll just make a suggestion here that you can accept if you want.
```suggestion
    A cone with the top cut off parallel to the cone bottom.
```